sdk/clientcore/annotation-processor/src/main/java/io/clientcore/annotation/processor/models/HttpRequestContext.java

```diff
@@ -290,6 +291,69 @@ public void setIsConvenience(boolean isConvenience) {
         this.isConvenience = isConvenience;
     }
 
+    /**
+     * Sets the static headers from an array of strings.
+     *
+     * @param headers the array of headers to set.
+     */
+    public void setHeaders(String[] headers) {
```

Comment:
I think this and the query param method should be called add as they both perform additive operations, not setting operations.